.changeset/tall-garlics-change.md

```diff
@@ -0,0 +1,11 @@
+---
+"@razorpay/blade": major
```

Comment:
So based on discussion during sprint planning we can make this minor right? agree with Kamlesh that it might not exactly be a breaking change since we're fixing it according to the existing designs.

Reply:
Done in https://github.com/razorpay/blade/pull/761/commits/e35ad717133247c3e00611f6fb5a3623efdb3ed4